infra/gcp/ensure-main-project.sh

```diff
@@ -136,6 +136,18 @@ empower_ksa_to_svcacct \
   "${PROJECT}" \
   $(svc_acct_email "${PROJECT}" "k8s-infra-gcp-auditor")
 
+color 6 "Ensuring the k8s-infra-dns-updater serviceaccount exists"
+ensure_service_account \
+  "${PROJECT}" \
+  "k8s-infra-dns-updater" \
+  "k8s-infra dns updater"
+
+color 6 "Empowering k8s-infra-dns-updater serviceaccount to be used on build cluster"
+empower_ksa_to_svcacct \
+    "k8s-prow.svc.id.goog[test-pods/k8s-infra-dns-updater]" \
```

Comment:
let's start on our trusted build cluster instead
```suggestion
    "k8s-infra-prow-build-trusted.svc.id.goog[test-pods/k8s-infra-dns-updater]" \
```

Reply:
Done